### ISSUE TITLE
Crash when try to open a v3.5 score in v3.6

### ISSUE THREAD

**Issue by PedroCuadra** [musescore/MuseScore#7299]
Hello everyone.

First of all, I'm not a developer of Musescore. I give my solution while this bug is active.

Recently, I tried to open a few Musescore 3.5 files, some of them crashed, some of them opened correcty. Here is my solution, for now:

- Make sure your last session is not restored.

## If you have a `.mscz` score or "Compressed musescore file":

- Open the file with **7zip** or **Winrar** or any other `zip`file viewer.
- Extract the `*.mscx` score to wherever you want. follow the next steps

## If you have a `.mscx` score or "Uncompressed musescore file":

At this step if I tried to open the file with Musescore it would crash again.

- Open the file with a text editor, like notepad or Visual Studio Code.
- Your score is going to be like this:
- 
```xml
<?xml version="1.0" encoding="UTF-8"?>
<museScore version="3.01">
  <programVersion>3.5.2</programVersion>
  <programRevision>465e7b6</programRevision>
  <Score>
    <layoutMode>line</layoutMode>
    <LayerTag id="0" tag="default"></LayerTag>
    <currentLayer>0</currentLayer>
    <Division>480</Division>
...
```

- Change

```xml
<layoutMode>line</layoutMode>
```

to

```xml
<layoutMode>page</layoutMode>
```

- Save the file and voilà, the file can be opened with Musescore

**Comment by MarcSabatella** [comment]
For the record, this is a known bug and that's a known workaround.  Fix will be in 3.6.1.  In the future, please use the Support forum for discussions - we don't use GitHub for this.

**Comment by jeetee** [comment]
See https://musescore.org/en/node/315529 which was closed by #7241
